komodo-relational/src/main/java/org/komodo/relational/teiid/CachedTeiid.java

```diff
@@ -72,10 +89,10 @@
          */
         @Override
         public CachedTeiid create( final UnitOfWork transaction,
-                             final Repository repository,
-                             final KomodoObject parent,
-                             final String id,
-                             final RelationalProperties properties ) throws KException {
+                                   final Repository repository,
+                                   final KomodoObject parent,
+                                   final String id,
+                                   final RelationalProperties properties ) throws KException {
             return new CachedTeiidImpl(transaction, repository, id);
```

Comment:
Sorry Barry, fundamentally disagree.

The Eclipse preferences will never get full agreement as they interfere with people's coding styles. For example, Dan prefers to leave gaps after his opening brackets while I do not. Mandating one or another is protractive and pointless since both are equally acceptable styles.

Likewise, committing reformatting changes when they are not required in a commit generates noise, distracting from the real code logic and architecture changes. Such noise makes it more time-consuming to review PRs and such changes contribute nothing to the overall code.